### PR TITLE
Blog App - VIEWS

### PR DESCRIPTION
**Pull request by misalazeem**
Following tasks were completed:

Implement the design from the sneak peek wireframes.
Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone)

### CODE REVIEW

File: app/views/shared/_user_photo.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<% if user.photo.present? %>
+    <div class="user-photo">
+      <%= image_tag user.photo, alt: user.name %>
+    </div>
+<% else %>
+    <div class="photo-placeholder">
+        No Photo uploaded
+    </div>
+<% end %>
```
Comment (KanzaTahreem) [inline app/views/shared/_user_photo.html.erb]: - [ ] Well done for using partials 👍👍 However, when I am using an image with the correct URL, this is how it is being displayed 👇 Kindly make sure to fix the width and height of the image according to the given wireframe. Also, please make sure to fix the image wherever it is used.

image

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+  </div>
+  <p><%= @post.text %></p>
+</div>
+<div class="comment-box">
+  <% @post.recent_five.each do |comment| %>
+    <div class="each-commet">
+      <span><%= comment.author.name %>: </span>
+      <span><%= comment.text %></span>
+    </div>
+  <% end %>
```
Comment (KanzaTahreem) [inline app/views/posts/show.html.erb]: - [ ] Please make sure that post show page displays post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) not only 5 comments.

Current | Expected 
---| ---|
image | image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @misalazeem 

Good job so far 👏! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Followed wireframe 
- ✅ Correct Linters
- ✅ Descriptive PR

## Required Changes 🛑

- [ ] Please ensure that we receive the user's index page when the app starts. Currently, Rails renders the default page. Please refer to the URL in the wireframe below 👇. You can achieve this by setting `root users#index` in `routes.rb`.

  image

_Please check the comments under the review._

## Optional suggestions ❕

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @misalazeem, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Required changes fixed ✔️
- Commit messages meaningful ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**